system/jlib/jptree.cpp

```diff
@@ -7903,10 +7892,14 @@ jlib_decl IPropertyTree * loadConfiguration(const char * defaultYaml, const char
             appendCurrentDirectory(fullpath, false);
             addNonEmptyPathSepChar(fullpath);
             fullpath.append(optConfig);
-            delta.setown(loadConfiguration(fullpath, componentTag));
+            delta.setown(loadConfiguration(fullpath, componentTag, true));
+            globalConfiguration.setown(loadConfiguration(fullpath, "global", false));
         }
         else
-            delta.setown(loadConfiguration(optConfig, componentTag));
+        {
+            delta.setown(loadConfiguration(optConfig, componentTag, true));
+            globalConfiguration.setown(loadConfiguration(optConfig, "global", false));
```

Comment:
Is there any reason why global should be optional?

Reply:
Is there any reason why it shouldn't ?

Unlikely to be an issue except when doing bits of local testing...